lib/aweplug/helpers/video/youtube.rb

```diff
@@ -46,7 +46,7 @@ def add(url, product: nil, push_to_searchisko: true)
                     :api_method => @youtube.videos.list,
                     :parameters => {
                       :id => v['snippet']['resourceId']['videoId'],
-                      :part => 'contentDetails'
+                      :part => 'contentDetails,statistics,player'
```

Comment:
I'm pretty sure that broke the build, but I'll check on that.  I remember having some build issues when I included all of them.  I think it was still part of the query though...I could be wrong